Operators/Types/lib/io/video/SpoutInput.cs

```diff
@@ -0,0 +1,329 @@
+using SharpDX.Direct3D11;
+using SharpDX.DXGI;
+using SpoutDX;
+using SpoutDX.__Symbols;
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using T3.Core;
+using T3.Core.Logging;
+using T3.Core.Operator;
+using T3.Core.Operator.Attributes;
+using T3.Core.Operator.Slots;
+using DeviceContext = OpenGL.DeviceContext;
+using Resource = SharpDX.DXGI.Resource;
+
+namespace T3.Operators.Types.Id_25307357_6f6c_45b1_a38d_de635510a845
+{
+    public class SpoutInput : Instance<SpoutInput>
+    {
+        [Output(Guid = "10955469-F5C0-476D-8A1B-9CB2803820A9", DirtyFlagTrigger = DirtyFlagTrigger.Animated)]
+        public readonly Slot<Texture2D> Texture = new();
+
+        public SpoutInput()
+        {
+            Texture.UpdateAction = Update;
+            _instance++;
+        }
+
+        private void Update(EvaluationContext context)
+        {
+            Command.GetValue(context);
```

Comment:
I believe this input is not required. The operator will be updated on every frame because of the DirtyFlagTrigger on the output. Could it be, that this is a left over from [SpoutOutput]?

Reply:
Oh, yes it is.